src/codeu/chat/client/commandline/Chat.java

```diff
@@ -323,7 +346,18 @@ public void invoke(List<String> args) {
           } else {
             panels.push(createConversationPanel(conversation));
 
-            //command convo-id (uuid of)convo-owner convo-title creation-time
+            // If this user isn't in the new conversations map, add them with a new Set
+            Set<Uuid> newConversations;
+            if (!newConversationsMap.containsKey(user.user.id)){
+              newConversations = new HashSet<>();
+            }
+            else { //if user already has Set of updated convos get it
+              newConversations = newConversationsMap.get(userPanelContext.user.id);
+            }
+
+            newConversations.add(conversation.conversation.id);
+            newConversationsMap.put(user.user.id, newConversations);
```

Comment:
This line can be deleted if you use `computeIfAbsent` because `HashMap`s are mutable.